Formula/auctex.rb

```diff
@@ -10,7 +10,7 @@ class Auctex < Formula
     depends_on "autoconf" => :build
   end
 
-  depends_on :tex
+  # depends_on :tex
```

Comment:
I still haven't decided what to do about this; since the formula won't build without a TeX installation.  We could either import a basic version of that requirement as well, or just remove the formula.

Reply:
I'm definitely an auctex fan. But I have also transitioned to using [`use-package`](https://github.com/jwiegley/use-package/) so my reliance on homebrew for emacs packages is diminishing.